deepchem/models/torch_models/robust_multitask.py

```diff
@@ -251,3 +253,131 @@ def _get_activation_class(self, activation_name: ActivationFn) -> Callable:
+            Alternatively this may be a single value instead of a list, in which case the same value is used for every layer.
+        activation_fns: list or object
+            the Tensorflow activation function to apply to each layer.  The length of this list should equal
+            len(layer_sizes).  Alternatively this may be a single value instead of a list, in which case the
+            same value is used for every layer.
+        n_classes: int
+            the number of classes
+        bypass_layer_sizes: list
+            the size of each dense layer in the bypass network. The length of this list determines the number of bypass layers.
+        bypass_weight_init_stddevs: list or float
+            the standard deviation of the distribution to use for weight initialization of bypass layers.
+            same requirements as weight_init_stddevs
+        bypass_bias_init_consts: list or float
+            the value to initialize the biases in bypass layers
+            same requirements as bias_init_consts
+        bypass_dropouts: list or float
+            the dropout probablity to use for bypass layers.
+            same requirements as dropouts
+        """
+        if not isinstance(activation_fns, nn.Module):
+            logger.warning(
+                "Warning: Activation functions should be of type nn.Module. Using default activation function: ReLU."
+            )
+            activation_fns = nn.ReLU()
+
+        # The labels are one-hot encoded.
+        loss = losses.SoftmaxCrossEntropy()
+        output_types = ['prediction', 'loss']
+        self.n_classes = n_classes
+        self.n_tasks = n_tasks
+
+        model = RobustMultitask(
+            n_tasks=self.n_tasks,
+            n_features=n_features,
+            layer_sizes=layer_sizes,
+            mode='classification',
+            weight_init_stddevs=weight_init_stddevs,
+            bias_init_consts=bias_init_consts,
+            weight_decay_penalty=weight_decay_penalty,
+            weight_decay_penalty_type=weight_decay_penalty_type,
+            activation_fns=activation_fns,
+            dropouts=dropouts,
+            n_classes=self.n_classes,
+            bypass_layer_sizes=bypass_layer_sizes,
+            bypass_weight_init_stddevs=bypass_weight_init_stddevs,
+            bypass_bias_init_consts=bypass_bias_init_consts,
+            bypass_dropouts=bypass_dropouts)
+        self.activation_fns = model.activation_fns
+        self.dropouts = model.dropouts
+        self.shared_layers = model.shared_layers
+        self.bypass_layers = model.bypass_layers
+        self.output_layers = model.output_layers
+
+        super(RobustMultitaskClassifier,
+              self).__init__(model,
+                             loss,
+                             output_types=output_types,
+                             regularization_loss=model.regularization_loss,
+                             **kwargs)
+
+    def default_generator(self,
```

Comment:
Docstring

Reply:
Done. Also added epoch training logger from the original method definition. Marking as resolved.